package.json

```diff
@@ -136,7 +136,6 @@
   },
   "scripts": {
     "vscode:prepublish": "npm run package",
-    "vsix": "vsce package --out bin",
```

Comment:
fwiw I don't think it hurts to leave this in case we want to build it manually for testing before pushing to the marketplace. Maybe we could just add a bin/.empty file to make sure the directory exists?

Reply:
Ok added back the build target, but removed the bin dependency and added roo-cline-.vsix to .gitignore